pkg/apis/componentconfig/types.go

```diff
@@ -896,6 +896,10 @@ type KubeControllerManagerConfiguration struct {
 	// through the kube-aggregator when enabled, instead of using the legacy metrics client
 	// through the API server proxy.
 	HorizontalPodAutoscalerUseRESTClients bool
+
+	// KeepTerminatedPodVolumes causes terminated pod volumes attached to the node after the pod terminates.
```

Comment:
=> KeepTerminatedPodVolumes keeps volumes attached to node after the pod terminates?